pom.xml

```diff
@@ -7,7 +7,7 @@
     <packaging>jar</packaging>
     <name>MaxMind GeoIP2 API</name>
     <description>GeoIP2 webservice client and database reader</description>
-    <url>http://dev.maxmind.com/geoip/geoip2/web-services</url>
+    <url>https://dev.maxmind.com/geoip/docs/web-services?lang=en</url>
```

Comment:
Do we have a page for both the webservices and databases? Since this package is for both.

Reply:
Sorta.  We have a landing page for GeoIP2 and GeoLite.  I'll use that